### ISSUE TITLE
Angular CLI 7: Loading chunk xxx failed on IE11

### ISSUE THREAD

**Issue by redplane** [angular/angular-cli#12986]
### Bug Report or Feature Request (mark with an `x`)
```
- [ X ] bug report -> please search issues before submitting
- [ ] feature request
```

### Command (mark with an `x`)
```
- [ X ] serve
```

### Versions
```
Angular CLI: 7.0.6
Node: 10.13.0
OS: win32 x64
Angular: 7.0.4
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, platform-server, router, upgrade

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.10.6
@angular-devkit/build-angular     0.10.6
@angular-devkit/build-optimizer   0.10.6
@angular-devkit/build-webpack     0.10.6
@angular-devkit/core              7.0.5
@angular-devkit/schematics        7.0.6
@angular/cli                      7.0.6
@ngtools/webpack                  7.0.6
@schematics/angular               7.0.6
@schematics/update                0.10.6
rxjs                              6.3.3
typescript                        3.1.6
webpack                           4.19.1
```

### Repro steps

When I run my application with command: `ng serve --ssl --host 0.0.0.0`.
1. The first module: **dashboard** is loaded successfully.
2. I filled information in a form of **dashboard** and navigate to **play game**, it still works.
3. In play game, when I navigated to **prize/win-game**, it threw an error on IE (works on Chrome):
**Loading chunk modules-prize-prize-module failed**

![error](https://user-images.githubusercontent.com/7974095/48669615-6edb4480-eb43-11e8-9b7a-b113db7d8c3d.png)

### Desired functionality
- Navigate to **prize/win-game** successfully.

### Mention any other details that might be useful

Here is my **app.route.component.ts**:

```
export const routes: Routes = [
  {
    path: '',
    children: [
      {
        path: '',
        pathMatch: 'full',
        redirectTo: '/dashboard'
      },
      {
        path: 'dashboard',
        loadChildren: 'modules/dashboard/dashboard.module#DashboardModule',
      },
      {
        path: 'play-game',
        loadChildren: 'modules/play-game/play-game.module#PlayGameModule',
        canActivate: [HasBillCodeGuard, HasNoPrizeGuard]
      },
      {
        path: 'support',
        loadChildren: 'modules/support/support.module#SupportModule',
      },
      {
        path: 'play-specially',
        loadChildren: 'modules/play-specially/play-specially.module#PlaySpeciallyModule',
        canActivate: [HasBillCodeGuard]
      },
      {
        path: 'store',
        loadChildren: 'modules/store/store.module#StoreModule',
      },
      {
        path: 'prize',
        loadChildren: 'modules/prize/prize.module#PrizeModule',
        canActivate: [HasBillCodeGuard]
      },
      {
        path: 'thank',
        loadChildren: 'modules/thank/thank.module#ThankModule',
      }
    ]
  },
  {
    path: 'master-layout',
    children: [
      {
        path: '',
        component: AuthorizeLayoutComponent
      }
    ]
  }
];

//#endregion

@NgModule({
  declarations: [
    AppComponent
  ],
  imports: [
    RouterModule.forRoot(routes, {enableTracing: false})
  ],
  exports: [
    RouterModule
  ],
  bootstrap: [AppComponent]
})

export class AppRouteModule {
}
```

This is my **prize.route.ts**

```
const routes: Routes = [
  {
    path: '',
    pathMatch: 'prefix',
    component: AuthorizeLayoutComponent,
    children: [
      {
        path: 'won-game',
        component: WinPrizeComponent,
        pathMatch: 'full',
        resolve: {
          availablePrizes: 'availablePrizeResolve'
        }
      },
      {
        path: 'lost-game',
        component: LostPrizeComponent,
        pathMatch: 'full'

      }
    ]
  }
];

@NgModule({
  imports: [
    RouterModule.forChild(routes)],
  exports: [RouterModule]
})
export class PrizeRouteModule {
}
```

**Comment by alan-agius4** [comment]
This seems like a bug but we'll need to look at a reproduction to find and fix the problem. Can you setup a minimal repro please?

You can read [here](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue) why this is needed. A good way to make a minimal repro is to create a new app via `ng new repro-app` and adding the minimum possible code to show the problem. Then you can push this repository to github and link it here.

This might be related to your directory structure so its really important to get an accurate repro to diagnose this.

**Comment by redplane** [comment]
Sorry for my late reply.
Actually it is very weird that the application can start, but when it goes to `play-game` screen, that error happened.

After having time troubleshooting the error, I realized that in `play-game` screen, I used [Firework canvas](https://github.com/tswaters/fireworks) that only support ES6.  When I removed that library declaration from my `play-game` screen, my app worked.

I have contacted the library author to ask for es5 support and he agreed.
Hope this information helps someone who faces with the same error.
